src/test/java/com/jwplayer/southpaw/serde/AvroSerdeTest.java

```diff
@@ -67,7 +68,7 @@ public void cleanup() {
         serde.close();
     }
 
-    @Test(expected = SerializationException.class)
+    @Test(expected = BufferUnderflowException.class)
```

Comment:
this potentially doesn't look like a good change although i need to look into the underlying implementation to see why it would throw a `BufferUnderflowException`

Reply:
after spending some time looking into this it doesnt seem clear but makes sense why it throws this exception.

disregard